qpong/containers/vbox.py

```diff
@@ -18,17 +20,25 @@
 
 
 class VBox(pygame.sprite.RenderPlain):
-    """Arranges sprites vertically"""
-    def __init__(self, xpos, ypos, *sprites):
+    """
+    Arranges sprites vertically
+    """
+
+    def __init__(self, xpos, ypos, orientation, *sprites):
```

Comment:
Are you trying to use orientation so that we can just use one VBOX to replace both VBOX and HBOX?

Reply:
Removed the changes and revert back to original.

Reply:
Many thanks for reviewing the PR, and subsequent modifications . I will try to closely follow your advise in subsequent PRs for a clear separation of changes and logic.

The changes to VBox and HBox were not meant to make it in the PR (sorry about that). I also noticed that HBox isn't used at all, hence to appease pylint, I meant revert back VBox to its original state and remove HBox entirely.